pom.xml

```diff
@@ -7,7 +7,7 @@
     <groupId>g-shock-team.robot</groupId>
     <artifactId>robot-shock-robot</artifactId>
     <packaging>pom</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <version>${project.version}</version>
```

Comment:
Fixed - you can check this in the next commit. This should be constant value. If not, then maven generates a lot of warnings :)